HISTORY.md

```diff
@@ -4,6 +4,11 @@
 * Fix a bug where range tombstone blocks in ingested files were cached incorrectly during ingestion. If range tombstones were read from those incorrectly cached blocks, the keys they covered would be exposed.
 * Fix a data race that might cause crash when calling DB::GetCreationTimeOfOldestFile() by a small chance. The bug was introduced in 6.6 Release.
 
+### Performance Improvements
+* In CompactRange, for levels starting from 0, if the level does not have any file with any key
+falling in the specified range, the level is skipped. So instead of always compacting from level 0,
+the compaction starts from the first level with keys in the specified range until the last such level.
```

Comment:
It looks like that we usually don't wrap lines in HISTORY.md. Can you follow this convention?